memdown.js

```diff
@@ -136,6 +136,7 @@ MemDOWN.prototype._open = function (options, callback) {
 }
 
 MemDOWN.prototype._put = function (key, value, options, callback) {
+  if (typeof value === 'undefined' || value === null) value = ''
```

Comment:
if a value is put as `false` it is expected to be reat as `"false"`, see https://github.com/Level/abstract-leveldown/blob/master/abstract/put-get-del-test.js#L134 (which compares `"false"` to `(false).toString()`)